TypeScript error in nivo/line: Module '"@nivo/core"' has no exported member 'DataFormatter'

**Describe/explain the bug**
I'm getting the following Typescript error with version "0.67.0":
Error:
```
node_modules/@nivo/tooltip/dist/types/BasicTooltip.d.ts:2:10 - error TS2305: Module '"@nivo/core"' has no exported member 'DataFormatter'.

2 import { DataFormatter } from '@nivo/core';
           ~~~~~~~~~~~~~

node_modules/@nivo/line/index.d.ts:17:5 - error TS2305: Module '"@nivo/core"' has no exported member 'DataFormatter'.

17     DataFormatter,
       ~~~~~~~~~~~~~

```

It seems like this commit (https://github.com/plouc/nivo/commit/5b158797da79430260c1e75165dd1c0cd15264e1) missed the files referenced in the error above.

**To Reproduce**
```
"@nivo/bar": "^0.67.0",
"@nivo/core": "^0.67.0",
"@nivo/line": "^0.67.0",
"@nivo/pie": "^0.67.0 ",
"@nivo/tooltip": "^0.67.0",
```

**Expected behavior**
No TypeScript error.

**Screenshots**
N/A

**Additional context**
TypeScript version 4.0.5

--- a/packages/line/index.d.ts
+++ b/packages/line/index.d.ts
@@ -14,7 +14,7 @@ import {
     MotionProps,
     CartesianMarkerProps,
     SvgDefsAndFill,
-    DataFormatter,
+    ValueFormat,
     DatumValue as CoreDatumValue,
 } from '@nivo/core'
 import { OrdinalColorScaleConfig } from '@nivo/colors'
@@ -133,9 +133,9 @@ declare module '@nivo/line' {
         data: Serie[]
 
         xScale?: Scale
-        xFormat?: string | DataFormatter
+        xFormat?: ValueFormat<DatumValue>
         yScale?: Scale
-        yFormat?: string | DataFormatter
+        yFormat?: ValueFormat<DatumValue>
 
         layers?: Layer[]
 
@@ -193,7 +193,7 @@ declare module '@nivo/line' {
         debugSlices?: boolean
         sliceTooltip?: SliceTooltip
 
-        tooltipFormat?: DataFormatter | string
+        tooltipFormat?: ValueFormat<DatumValue>
         tooltip?: PointTooltip
 
         enableCrosshair?: boolean
@@ -221,7 +221,7 @@ declare module '@nivo/line' {
     export type CustomCanvasLayer = (props: CustomCanvasLayerProps) => void
     export type CanvasLayer = LineLayerType | CustomCanvasLayer
 
-    export interface LineCanvasProps extends LineProps {
+    export interface LineCanvasProps extends Omit<LineProps, 'layers'> {
         pixelRatio?: number
         layers?: CanvasLayer[]
     }

